src/main/java/frc/robot/TestMode.java

```diff
@@ -55,7 +58,16 @@ public TestMode(
     this.climbSubsystem = climbSubsystem;
     this.armSubsystem = armSubsystem;
     this.indexerSubsystem = indexerSubsystem;
-  }
+
+    testResults = new BooleanSubscriber[] {
+        indexerForwardPublisher.getTopic().subscribe(false),
+        indexerBackwardPublisher.getTopic().subscribe(false),
+        manipulatorForwardPublisher.getTopic().subscribe(false),
+        manipulatorBackwardPublisher.getTopic().subscribe(false),
+        climbPublisher.getTopic().subscribe(false),
+        elevatorPubliser.getTopic().subscribe(false),
+        armPublisher.getTopic().subscribe(false) };
```

Comment:
It's kind of weird that this class is using NetworkTables to send state to itself, but I suppose it works.